### ISSUE TITLE
Kyverno CPU resource limits too low

### ISSUE THREAD

**Issue by brianrexrode** [defenseunicorns/uds-package-dubbd#379]
While doing some recent testing on DUBBD deployments, we ran into issues with kyverno pod resource limits being exceeded and causing pod restarts which in turn is causing other BB pods to be stuck in init status with the following errors when doing a describe on the pods:
```
Failed to create pod sandbox: rpc error: code = Unknown desc = failed to setup network for sandbox "5ba287e76728b49f7ce99940d01d6b8e274b35c14a7c0daa2a531c3a067383e8": plugin type="aws-cni" name="aws-cni" failed (add): add cmd: failed to assign an IP address to container
```
In some cases these errors clear themselves and some pods finally initialize and become ready. In other cases the pods remain in init status ultimately causing the associated helmrelease to fail which eventually causes the zarf pkg deploy to fail. Also found numerous errors like this having issues communicating with kyverno svc:
```
Internal error occurred: failed calling webhook \"validate.kyverno.svc-fail\": failed to call webhook: Post \"[https://kyverno-kyverno-svc.kyverno.svc:443/validate/fail?timeout=30s](https://kyverno-kyverno-svc.kyverno.svc/validate/fail?timeout=30s)\": context deadline exceeded"}
```
I was able to manually increase the cpu/memory resource limits for kyverno (I doubled them) and tested the deployment again to ensure all pods came up. While running kubectl top pods -n kyverno we noticed that the pods CPU usage was upwards of 1000m (BB charts default to 500m). 
Found this [issue](https://github.com/kyverno/kyverno/issues/5485#issuecomment-1422468133) which identified a memory leak in kyverno v1.8.5.

**Comment by brianrexrode** [comment]
It looks like BB 2.6.0 will include the kyverno upgrade to v1.9.2 hopefully fixing the memory leak issue

**Comment by oates** [comment]
@MxNxPx you got this one or tracking?

**Comment by brianrexrode** [comment]
I was able to test a deployment of DUBBD that included BB 2.6.0 and can confirm the deployment did complete successfully. kyverno pods did experience some CPU spikes but decreased CPU usage quickly. 

I would consider this issue `Closed` as soon as a new release is cut that includes BB 2.6.0.

Thanks

**Comment by MxNxPx** [comment]
closing as [this dubbd release](https://github.com/defenseunicorns/uds-package-dubbd/pull/400) includes BB 2.6.0
